lib/fs.js

```diff
@@ -2379,8 +2379,13 @@ WriteStream.prototype.open = function() {
 
 
 WriteStream.prototype._write = function(data, encoding, cb) {
-  if (!(data instanceof Buffer))
-    return this.emit('error', new Error('Invalid data'));
+  if (!(data instanceof Buffer)) {
+    const err = new errors.TypeError('ERR_INVALID_ARG_TYPE',
+                                     'data',
+                                     'Buffer',
+                                     data);
+    return this.detroy(err);
```

Comment:
`detroy` -> `destroy`

Reply:
@starkwang Oops 😬 That’s embarrassing. 
I blame @mcollina because I copied the comment above verbatim 😜

In other news, I realized my MacBook wasn’t actually building because Xcode isn’t installed, but linking seems to work fine. 🤨

Let’s see if this one works 💯